### PR TITLE
Kats/cer-1885-templating-docs

### PR DESCRIPTION
**Pull request by Katsie011**

### CODE REVIEW

File: cerebrium/fine-tuning/diffusion-models/config.mdx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+In your YAML file, you need to include the following required parameters or parse them into the `cerebrium train` command:
+
+### Required parameters
+| Parameter               | Suggested Value                | Description                                                                          |
+| ----------------------- | ------------------------------ | ------------------------------------------------------------------------------------ |
+| training_type           | diffuser                       | Type of training to run. Either diffuser or transformer. In this case, use diffuser. |
+| name                    | test-config-file               | Name of the experiment.                                                              |
+| api-key                 | Your private API key           | Your Cerebrium API key.                                                              |
+| hf_model_path           | runwayml/stable-diffusion-v1-5 | ---                                                                                  |
+| train_prompt            | Photo of a tsdf dog.           | ---                                                                                  |
```
Comment (jonoirwinrsa) [inline cerebrium/fine-tuning/diffusion-models/config.mdx:28]: could maybe explain tsdf in the description?

File: cerebrium/fine-tuning/diffusion-models/config.mdx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+Your yaml config file can be placed anywhere on your system, just point the trainer to the file.
+
+In your YAML file, you need to include the following required parameters or parse them into the `cerebrium train` command:
+
+### Required parameters
+| Parameter               | Suggested Value                | Description                                                                          |
+| ----------------------- | ------------------------------ | ------------------------------------------------------------------------------------ |
+| training_type           | diffuser                       | Type of training to run. Either diffuser or transformer. In this case, use diffuser. |
+| name                    | test-config-file               | Name of the experiment.                                                              |
+| api-key                 | Your private API key           | Your Cerebrium API key.                                                              |
```
Comment (jonoirwinrsa) [inline cerebrium/fine-tuning/diffusion-models/config.mdx:26]: on the call we mentioned this was optional but its under required params?

File: cerebrium/fine-tuning/diffusion-models/config.mdx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+| log_level               | INFO                           | log_level level for logging. Can be DEBUG, INFO, WARNING, ERROR.                     |
+| train_image_dir         | data/training_class_images/    | Directory of training images.                                                        |
+
+#### Optional parameters
+
+Additionally, you can include the following parameters in your config file:
+
+| Parameter                        | Suggested Value         | Description                                                                |
+| -------------------------------- | ----------------------- | -------------------------------------------------------------------------- |
+| **Diffuser Parameters** |                         |                                                                            |
```
Comment (jonoirwinrsa) [inline cerebrium/fine-tuning/diffusion-models/config.mdx:38]: spacing of the |'s if off on this line

Comment (jonoirwinrsa) [reply cerebrium/fine-tuning/diffusion-models/config.mdx:38]: does it matter?

File: cerebrium/fine-tuning/diffusion-models/config.mdx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+|               | gradient_accumulation_steps       | 1               | Number of gradient accumulation steps.                                                                             |
+|               | learning_rate                     | 0.0005          | The learning rate you would like to train with. Can be more aggressive than you would use to train a full network. |
+|               | lr_scheduler                      | constant        | Learning rate schedule. Can be one of ["constant", "linear", "cosine", "polynomial"]                               |
+|               | lr_warmup_steps                   | 5               | Number of learning rate warmup steps.                                                                              |
+|               | lr_num_cycles                     | 1               | Number of learning rate cycles to use.                                                                             |
+|               | lr_power                          | 1.0             | Power factor if using a spolynomial scheduler.                                                                     |
```
Comment (jonoirwinrsa) [inline cerebrium/fine-tuning/diffusion-models/config.mdx:70]: spolynomial or polynomial?

File: cerebrium/fine-tuning/diffusion-models/dataset.mdx [hunk 5]
```diff
@@ -0,0 +1,75 @@
+---
+title: "Dataset Curation"
+description: "Guide to curating a dataset for success when fine-tuning your diffusion model on Cerebrium."
+---
+
+<Note>
+  Cerebrium's fine-tuning functionality is in public beta and we are adding
+  more functionality each week! If there are any issues or if you have an urgent requirement, please reach
+  out to [support](mailto\:support@cerebrium.ai)
```
Comment (jonoirwinrsa) [inline cerebrium/fine-tuning/diffusion-models/dataset.mdx:9]: I don't think it should have that slash in there? 🤔

File: cerebrium/fine-tuning/diffusion-models/config.mdx (hunk 1 above)
Comment (Katsie011) [reply cerebrium/fine-tuning/diffusion-models/config.mdx:28]: In hindsight that probably shouldn't be in there.

File: cerebrium/fine-tuning/diffusion-models/config.mdx (hunk 4 above)
Comment (Katsie011) [reply cerebrium/fine-tuning/diffusion-models/config.mdx:70]: Thanks

File: cerebrium/fine-tuning/diffusion-models/config.mdx (hunk 3 above)
Comment (Katsie011) [reply cerebrium/fine-tuning/diffusion-models/config.mdx:38]: Doesn't matter but I'll fix for readability

File: cerebrium/fine-tuning/diffusion-models/dataset.mdx (hunk 5 above)
Comment (Katsie011) [reply cerebrium/fine-tuning/diffusion-models/dataset.mdx:9]: Thanks. Didn't check that one. There were a couple others like that too. Good save.

File: cerebrium/fine-tuning/language-models/config.mdx [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+|                     | load_in_8bit                | True                 | Whether to load in the model in 8bit.                                  |
+|                     | device_map                  | "auto"               | Device map for loading in the model.                                   |
+| **peft_lora_args**  |                             |                      | Peft lora kwargs for use by _PeftConfig()_                             |
+|                     | r                           | 8                    | The r value for LoRA.                                                  |
+|                     | lora_alpha                  | 32                   | The lora_alpha value for LoRA.                                         |
+|                     | lora_dropout                | 0.05                 | The lora_dropout value for LoRA.                                       |
+|                     | target_modules              | ["q_proj", "v_proj"] | The target_modules for LoRA. These are the suggeseted values for Llama |
```
Comment (jonoirwinrsa) [inline cerebrium/fine-tuning/language-models/config.mdx:62]: @Katsie011 spelling suggeseted

File: cerebrium/fine-tuning/diffusion-models/dataset.mdx [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+- Images should be sized to 512 x 512 for stable diffusion v1.5. 
+  - If not, we'll resize them for you, but this may result in a loss of quality.
+- Your images should all be upright or in the same orientation.
+- Keep the number of training images low. Aim for between 10 and 15 as your model may not converge if you use too many.
+- Make sure your images are of the same class. For example, if you are fine-tuning the prompt "Photo of a dog", all of your images should be of dogs.
+
+
+
+If you are doing style-based fine-tuning:
+- Try fine-tuning a single style at a time. Keep the style consistent across all of your images.
```
Comment (jonoirwinrsa) [inline cerebrium/fine-tuning/diffusion-models/dataset.mdx:27]: so minor but could be 3 points:

- Fine-tuning a single style at a time
- Keep the style consistent across all of your images
- Keep the colouring consistent across all of your images

**Review by jonoirwinrsa — APPROVED** [review]
2 smol comments, otherwise I am happy

File: cerebrium/fine-tuning/language-models/config.mdx (hunk 6 above)
Comment (Katsie011) [reply cerebrium/fine-tuning/language-models/config.mdx]: Thanks @jonoirwinrsa

**Pull request merged**